tests/route/test_route_perf.py

```diff
@@ -60,6 +60,10 @@ def check_config(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_rand_
         return
 
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    if (duthost.facts.get('platform_asic') == 'broadcom-dnx'):
```

Comment:
Can we add this is the [test_mark_conditions.yaml](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml)?

Reply:
It's just the fixture `check_config` that this PR is skipping, the rest of the test still runs and passes on DNX SKUs.
If we were to add this condition in `test_mark_conditions.yaml` it would skip the whole test right?

Reply:
yes, I think current fix is ok. We only want to skip this newly added fixture for dnx sai because `conf show l3_alpm_enable` is not working.
otherwise the test should be run